assets/js/modules/search-console/components/dashboard/DashboardImpressionsWidget.js

```diff
@@ -61,6 +63,8 @@ function DashboardImpressionsWidget() {
 		if ( url ) {
```

Comment:
There's a problem here that I just noticed, which was missed during code review and QA of #1923: The above value for `num_of_days` is invalid (e.g. `28 days` instead of expected `28`), and the call to `getCurrentDateRange` is incorrect (it doesn't support a second parameter).

Unrelated to this PR, but we should fix it.